### PR TITLE
Doc changes sample apps - Add 3 apps and change installation.md flow

### PR DESCRIPTION
**Pull request by TvisharajiK**
# Pull Request Template

## Description

Please include a summary of the change. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Additional Context (Please include any Screenshots/gifs if relevant)

...

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding tests.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by Sonichigo** [comment]
@TvisharajiK Deploy has failed please check once via npm run build locally to test the changes

File: versioned_docs/version-2.0.0/quickstart/java-spring-postgres.md [hunk 1]
```diff
@@ -130,17 +132,25 @@ Here we just need to change the command used to start the application.
 keploy record -c "docker compose up" --container-name javaApp --build-delay 100
 ```
 
+<img src="/docs/img/pet-clinic-record.png" alt="Sample Keploy Test Result Node JWT" width="100%" style={{ borderRadius: '5px' }} />
```
Comment (Sonichigo) [inline versioned_docs/version-2.0.0/quickstart/java-spring-postgres.md:135]: alt tags needs to be updated here and above too.

File: versioned_docs/version-2.0.0/quickstart/sample-ts.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+We will be using Docker compose to run the application as well as GraphQL on Docker container.
+
+### Capture the testcases
+
+We will run the keploy in record mode with docker-compose to start our application:-
+
+```bash
+keploy record -c "sudo docker-compose up" --containerName "ts-nhost"
+```
+<img src="/docs/img/wsl-record-ts.png" alt="Sample Keploy Test Result Node JWT" width="100%" style={{ borderRadius: '5px' }} />
```
Comment (Sonichigo) [inline versioned_docs/version-2.0.0/quickstart/sample-ts.md:215]: alt tag value need to be updated

File: versioned_docs/version-2.0.0/quickstart/sample-ts.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+Now, let's put things to test and run the keploy in test mode again:-
+
+```bash
+sudo -E env "PATH=$PATH" keploy test -c 'ts-node src/app.ts' --delay 10
+```
+Voila!! Our test cases have passed 🌟
+
+You will be able to see the summary of your test cases run in your terminal!
+
+<!-- ![Test Record](/img/wsl-test-ts.png)
```
Comment (Sonichigo) [inline versioned_docs/version-2.0.0/quickstart/sample-ts.md:198]: this can be removed if not being used

**Review by Sonichigo — CHANGES_REQUESTED** [review]

File: versioned_docs/version-2.0.0/quickstart/sample-ts.md [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+
+Let's run our captured test cases
+
+```bash 
+keploy test -c 'sudo docker-compose up' --containerName "ts-nhost" --delay 10
+```
+This is what your response should look like!
+
+<img src="/docs/img/wsl-test-ts.png" alt="Sample Keploy Test TS Nhost" width="100%" style={{ borderRadius: '5px' }} />
+
```
Comment (Sonichigo) [inline versioned_docs/version-2.0.0/quickstart/sample-ts.md:267]: Two images are together. Please add a one line on how both are different.

File: versioned_docs/version-2.0.0/quickstart/java-spring-postgres.md [hunk 5]
```diff
@@ -130,17 +132,25 @@ Here we just need to change the command used to start the application.
...
+
 ## Running the testcases using Keploy
 
 ```bash
 keploy test -c "docker compose up" --container-name javaApp --build-delay 50 --delay 20
 ```
 
+<img src="/docs/img/pet-clinic-test1.png" alt="Sample Keploy Test Java" width="100%" style={{ borderRadius: '5px' }} />
+
+
```
Comment (Sonichigo) [inline versioned_docs/version-2.0.0/quickstart/java-spring-postgres.md:146]: Two images are together. Please add a one line on how both are different.

Comment (TvisharajiK) [reply versioned_docs/version-2.0.0/quickstart/java-spring-postgres.md:146]: hey any update on this? This was fixed

**Pull request merged**